Automattic/jetpack
# Instant Search: Support yarn watch

#### Changes proposed in this Pull Request:

* Splits the existing `build-search` script into `build-search:scripts` and `build-search:styles`.
* Adds support for watching `build-search:scripts` via `yarn watch`. `build-search:styles` is already being watched via `sass:instant-search` in `tools/builder/sass.js`.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* No.

#### Testing instructions:
* Run `yarn build-search`. Ensure assets build as expected. 
* Run `yarn watch` and try changing a script in `modules/search/instant-search`. Ensure that search assets are rebuilt.

#### Proposed changelog entry for your changes:
* None.
